src/utils/utils.js

```diff
@@ -0,0 +1,12 @@
+// Detects if the current browser is IE.
+const isIE = () => {
+  if (!NODE) {
```

Comment:
Let's add a comment here for clarity:
// this require call returns the running version of IE or undefined